pkg/process/procutil/process_linux.go

```diff
@@ -191,7 +191,7 @@ func (p *Probe) StatsForPIDs(pids []int32, now time.Time) (map[int32]*Stats, err
 }
 
 // ProcessesByPID returns a map of process info indexed by PID
-func (p *Probe) ProcessesByPID(now time.Time) (map[int32]*Process, error) {
+func (p *Probe) ProcessesByPID(now time.Time, collectStats bool) (map[int32]*Process, error) {
```

Comment:
Since we're extending the behavior of `ProcessesByPID`, should we add an unit test in `process_linux_test.go` to ensure that it's working as expected when `collectStats` is set to `false`?

Reply:
That's a really smart idea. I will create one :)